frontend/js/src/explore/artist-similarity/SimilarArtistsGraph.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect, useState } from "react";
+import {
+  ResponsiveNetwork,
```

Comment:
I'm not sure exactly how to use this properly, but for me currently the graph ends up cropped if I'm not at the expected screen size ratio:
<img width="883" alt="image" src="https://github.com/metabrainz/listenbrainz-server/assets/6179856/b718c39f-6f84-4b1e-96e6-6b99927ff2e0">